README.md

```diff
@@ -285,7 +285,9 @@ an exception.)
 
 Alternatively, you can define a system property `creaper.wildfly` (its value
 is ignored, the system property just needs to be defined). This switches
-the default protocol to `http-remoting` and the default port to `9990`.
+the default protocol to `http-remoting` and the default port to `9990`
+(in case `ssl` is used, this switches the default protocol to `https-remoting`
```

Comment:
Instead of "in case `ssl` is used", which kinda implies that there's another system property called "ssl", I'd say "in case SSL is configured".